sdk/storage/azure-storage-blob/src/main/java/com/azure/android/storage/blob/StorageBlobServiceImpl.java

```diff
@@ -75,12 +73,12 @@
         this.serializerAdapter = serviceClient.getSerializerAdapter();
     }
 
-    List<BlobItem> getBlobsInPage(String pageId,
-                                  String containerName,
-                                  ListBlobsOptions options) {
+    ListBlobsFlatSegmentResponse listBlobFlatSegment(String pageId,
```

Comment:
`listBlobs` maybe?

Reply:
it's protocol layer (hidden from user) and is how named in swagger, we can revisit this once azure-core is released, storage we have more time to clean up.